### ISSUE TITLE
Missing title attribute on iframe to comply with WCAG AA

### ISSUE THREAD

**Issue by LucGosso** [google/recaptcha#355]
Missing title attribute on recaptcha iframe. Must be localizable.

https://www.w3.org/TR/WCAG20-TECHS/H64.html

Also this is a legal requirement: European Web Accessibility Directive saying all goverment agencies (public or governmental entity) needs to implement WCAG. https://www.3playmedia.com/2019/03/29/the-european-web-accessibility-directive/ 

Version 3.

**Comment by jenfloods** [comment]
What is the resolution to get the missing title attribute on the recaptcha iframe? I am getting it error issue "iframe[title="reCAPTCHA"],html" <html jstcache="0">
Failure Details
Document does not have a non-empty <title> element

**Comment by rowan-m** [comment]
Closing old issues that are not related to the PHP client code.

**Comment by OVigilante** [comment]
> Missing title attribute on recaptcha iframe. Must be localizable.
> 
> https://www.w3.org/TR/WCAG20-TECHS/H64.html
> 
> Also this is a legal requirement: European Web Accessibility Directive saying all goverment agencies (public or governmental entity) needs to implement WCAG. https://www.3playmedia.com/2019/03/29/the-european-web-accessibility-directive/
> 
> Version 3.

Hello,
Why is this issue closed?
I don't see any fix.
This original issue continue to existe.
